exposed-tests/src/test/kotlin/org/jetbrains/exposed/sql/tests/sqlite/ForeignKeyConstraintTests.kt

```diff
@@ -184,7 +184,8 @@ class ForeignKeyConstraintTests : DatabaseTestsBase() {
             override val primaryKey = PrimaryKey(id)
         }
 
-        withTables(category, item) {
+        // TODO probably could be fixed for MySql 5
```

Comment:
I would log an issue for that and link it in a comment

Reply:
Yep, created [[EXPOSED-410](https://youtrack.jetbrains.com/issue/EXPOSED-410) MySql: update tests to support MySql8.](https://youtrack.jetbrains.com/issue/EXPOSED-410/MySql-update-tests-to-support-MySql8.)

I wanted to create it after merging of this PR (if it's merged), because there is a probability that a reason not to merge could be found.